src/tvheadend/HTSPDemuxer.cpp

```diff
@@ -149,7 +149,8 @@ DEMUX_PACKET* HTSPDemuxer::Read()
   }
   Logger::Log(LogLevel::LEVEL_TRACE, "demux read nothing");
 
-  if (m_lastPkt > 0 && m_lastUse - m_lastPkt > 10 && !IsPaused())
+  if (m_lastPkt > 0 && m_lastUse - m_lastPkt > m_settings->GetStreamStalledThreshold() &&
+      !IsPaused())
   {
     Logger::Log(LogLevel::LEVEL_WARNING,
                 "demux read no data for at least 10 secs; restarting connection");
```

Comment:
The log message may be wrong if the user changes the default

Reply:
Oops, thanks.